smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -142,6 +142,46 @@ contract MixinTransfer is
     ) {
       revert UNAUTHORIZED();
     }
+    _transferFrom(_from, _recipient, _tokenId);
+  }
+
+  /** 
+  * Lending a key allows you to transfer the token while retaining the 
+  * ownerships right by setting yourself as a key manager first
+  * @param _from the owner of token to transfer
+  * @param _recipient the address that will receive the token
+  * @param _tokenId the id of the token
+  * @notice This requires the key manager to be set 
+  */
+
+  function lendKey(
+    address _from,
+    address _recipient,
+    uint _tokenId
+  )
+    public
+  {
+    _isValidKey(_tokenId);
+    _onlyKeyManagerOrApproved(_tokenId);
```

Comment:
Let's not use `approvals` here for simplicity? Only the key manager should be able to lend a key,

Reply:
Actually let's make things simple:
* LendKey can only be called by the KeyManager, OR
* if key manager is not set, by the key owner and, in that case, it should set the keyManager to be the current owner (not the one to which the key is transferred)

Let's also make sure we have a function to *unset* the key manager (or, alternatively, change the transfer to function to allow transfers even if there is a key manager, but in that case, reset the key manager).

wdyt?

Reply:
yep, better to remove approvals altogether and simplify the key manager logic